spec/aruba/fixtures/railsapp/config/initializers/assets.rb

```diff
@@ -1,3 +1,4 @@
 ## this file overrides the default initialization of assets.rb
 
- Rails.application.config.assets.version = "1.0" if Rails.version < '7.0'
\ No newline at end of file
+ Rails.application.config.assets.version = "1.0" if Rails.version < '7.0'
+ 
```

Comment:
there is still a missing new line at the end of the file, this may be some editor configuration

![image](https://user-images.githubusercontent.com/556268/141982222-7ad39c95-367a-420b-80a1-8df634d958b3.png)

the file should not end with the warning